Headlines/src/DataSources/ProfileDataSource.swift

```diff
@@ -12,75 +12,77 @@ import UIKit
     let reactionsService = ReactionsService()
     let interestsService = InterestsService()
-    
+
     var reactions = [Reaction]()
     var interests = [Interest]()
     var error: Error?
-    
+
     func fetchProfileData (success: ((_ interests: [Interest], _ reactions: [Reaction]) -> Void)?,
                            fail: ((_ error: Error) -> Void)?) {
-        
+
         let group = DispatchGroup()
         let queue = DispatchQueue(label: "profile data source")
-        
-        let reactionsSuccess: ((URLResponse?, [Reaction]) -> Void) = { [unowned self] (response, reactions) in
+
+        let reactionsSuccess: ((URLResponse?, [Reaction]) -> Void) = { [unowned self] response, reactions in
             self.reactions.removeAll()
             self.reactions.append(contentsOf: reactions)
             group.leave()
         }
-        
-        let interestSuccess: ((URLResponse?, [Interest]) -> Void) = { [unowned self] (response, interests) in
+
+        let interestSuccess: ((URLResponse?, [Interest]) -> Void) = { [unowned self] response, interests in
             self.interests.removeAll()
             self.interests.append(contentsOf: interests)
             group.leave()
         }
-        
-        let serviceFail: ((Error) -> Void) = { (error) in
+
+        let serviceFail: ((Error) -> Void) = { error in
             self.error = error
             group.leave()
         }
-        
+
         group.enter()
         reactionsService.getReactions(success: reactionsSuccess, fail: serviceFail)
-        
+
         group.enter()
         interestsService.getInterests(success: interestSuccess, fail: serviceFail)
-        
+
         group.notify(queue: queue) { [unowned self] in
             if let error = self.error, let fail = fail {
                 DispatchQueue.main.async {
                     fail(error)
                 }
                 return
             }
-            
+
             if let success = success {
                 DispatchQueue.main.async {
                     success(self.interests, self.reactions)
                 }
             }
         }
     }
-        
+
     // MARK: UICollectionViewDataSource
     func numberOfSections(in collectionView: UICollectionView) -> Int {
         return 2
     }
-    
+
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         switch section {
         case 0:
             return interests.count
+
         case 1:
             return reactions.count
+
         default:
             return 0
         }
     }
-    
+
     func collectionView(_ collectionView: UICollectionView,
                         cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        
+
         switch indexPath.section {
         case 0:
             guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "interest_cell", for: indexPath) as? LabelCollectionViewCell else {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f5a5ab936ee0d164c160af7374b637afef6eb821/Headlines/src/DataSources/ProfileDataSource.swift#L88"></span>Line should be 120 characters or less: currently 152 characters<br />
<code>line_length</code> <code>ProfileDataSource.swift:88</code></td>
    </tr>
  </tbody>
</table>